WooCommerce/Classes/ViewRelated/Products/Cells/ProductReviewsTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import Gridicons
+
+
+// MARK: - ProductReviewsTableViewCell
+//
+class ProductReviewsTableViewCell: UITableViewCell {
```

Comment:
Would you consider making this class final? (unless, of course, you are planning on subclassing it already)